docs/root/version_history/current.rst

```diff
@@ -11,6 +11,7 @@ Minor Behavior Changes
 ----------------------
 *Changes that may cause incompatibilities for some users, but should not for most*
 
+* local_ratelimit: local_ratelimit will consume tokens of all matched descriptors. This behavioral change can be reverted by setting runtime guard ``envoy.reloadable_features.http_local_ratelimit_match_all_descriptors`` to false.
```

Comment:
Can you also update the documentation of both local rate limit filters so it's clear what this new behavior is? You could edit the field docs of the filter APIs?

Reply:
Sorry, do you mean that  I should update the docs of both http and network local rate limit filter APIs?

Reply:
Sorry, do you mean that I should update the docs of both http and network local rate limit filter APIs? @mattklein123

Reply:
Yes please.